index.html

```diff
@@ -0,0 +1,82 @@
+            <p>Address : 16/2 Soi Ratchaphruek9, Taling-Chan, Bangkok, 10170</p>
+            <p>Phone : 092-241-4232</p>
+         </div>
+      </div>
+      <div class="background-detail">
+         <div class="experience">
+            <div class="left">
+               <span>Work Experience</span>
+            </div>
+            <div class="right">
+               <div class="group-p-header">
+                  <p class="p-header"> CABIN CREW</p>
+                  <p class="p-header"> MAY 2015 – 2020, ASIANA AIRLINES</p>
+               </div>
+               <ul>
+                  <li> Provided excellent 5-star airline service to the passenger.</li>
+                  <li> Lead the junior crews to organize the galley and ensure meals, items and services are correctly served and satisfied the passengers.</li>
+                  <li> Collaborated with multinational crews on the flight .</li>
+                  <li> Managed passengers’ emotions and expectations under pressure situations. </li>
+                  <li> Create in-flight service summary and report to the manager.</li>
+               </ul>
+               <div class="group-p-header">
+                  <p class="p-header">ASSISTANT SALES MANAGER</p>
+                  <p class="p-header">APR 2014 – APR 2015, JP FOODS EXPORT LTD.</p>
+               </div>
+               <ul>
+                  <li> Supported overseas customers to place orders via phone calls and email and managed the orders in the ERP system.</li>
+                  <li> Provided product information including price, quality, product differentiation to the overseas customer’s inquiries via phone call and email.</li>
+                  <li> Coordinated with the UK warehouse staff and carrier service provider to expedite product delivery and inform the delivery status to the customers.</li>
+                  <li> Followed up payment from the customers regarding the given credit terms.</li>
+                  <li> Followed up on sales problems and provided professional services to the customers.</li>
+               </ul>
+            </div>
+         </div>
+         <div class="education">
+            <div class="left">
+               <span>Education</span>
+            </div>
+            <div class="right">
+               <div class="group-p-header">
+                  <p class="p-header">2010 - 2014</p>
+                  <p> THAMMASAT UNIVERSITY, BACHELOR’S DEGREE,</p>
+                  <p> LIBERAL ARTS (RUSSIAN STUDY PROGRAM)</p>
+               </div>
+            </div>
+         </div>
+         <div class="skills">
+            <div class="left">
+               <span>Skills</span> 
+            </div>
+            <div class="right">
+               <ul>
+                  <li>HTML</li>
+                  <li>CSS</li>
+                  <li>English</li>
+                  <li>Korean</li>
+               </ul>
+            </div>
+         </div>
+      </div>
+      <div class="bottom"></div>
```

Comment:
Maybe better to name it `footer` ?
```suggestion
      <div class="footer"></div>
```